dts/arm/nordic/nrf54l15_cpuapp_peripherals.dtsi

```diff
@@ -72,6 +72,16 @@ timer10: timer@85000 {
 	prescaler = <0>;
 };
 
+rtc10: rtc@86000 {
+	compatible = "nordic,nrf-rtc";
+	reg = <0x86000 0x1000>;
```

Comment:
Could you also add corresponding entries in `validate_base_addresses.c`?

Reply:
Added.